### PR TITLE
EPIC: B1 - Backend endpoints to toggle (PRIORITY)

### PR DESCRIPTION
**Pull request by bella-ward**
This PR:

Adds POST endpoint for toggleAdmin and tests.
Adds POST endpoint for toggleInstructor and tests.
Adds ROLE_INSTRUCTOR to securityConfig.
Passes test coverage.

Dokku deployment: https://organic-bella-dev.dokku-07.cs.ucsb.edu/ 

- add /swagger-ui/index.html to get to swagger AFTER logging in

Closes https://github.com/ucsb-cs156-f23/proj-organic-f23-6pm-3/issues/9

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/organic/entities/User.java [hunk 1]
```diff
@@ -10,6 +10,16 @@
...
 
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+// import javax.persistence.GeneratedValue;
+// import javax.persistence.GenerationType;
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/organic/entities/User.java:21]:
```suggestion

```

File: src/main/java/edu/ucsb/cs156/organic/entities/User.java [hunk 2]
```diff
@@ -10,6 +10,16 @@
...
 import java.util.List;
 
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+// import javax.persistence.GeneratedValue;
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/organic/entities/User.java:20]:
```suggestion

```

**Review by pconrad — CHANGES_REQUESTED** [review]
Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by Garrett-Hu — APPROVED** [review]
lgtm, solid tests

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**